src/test/java/spoon/test/compilationunit/TestCompilationUnit.java

```diff
@@ -90,11 +97,11 @@ public void testGetUnitTypeWorksWithCreatedObjects() {
 		CtPackage myPackage = launcher.getFactory().Package().getOrCreate("my.package");
 		CompilationUnit cu = launcher.getFactory().createCompilationUnit();
 		assertEquals(CompilationUnit.UNIT_TYPE.UNKNOWN, cu.getUnitType());
-
-		cu.setDeclaredPackage(myPackage);
```

Comment:
could we stay backward compatible here and keep setDeclaredPackage?

Reply:
yes, this can be derived method which will internally create a package reference and package declaration. I will do so